src/engine/shanghai.md

```diff
@@ -142,4 +143,4 @@ This method follows the same specification as [`engine_forkchoiceUpdatedV1`](./p
 
 This method follows the same specification as [`engine_getPayloadV1`](./paris.md#engine_getpayloadv1) with the addition of the following:
 
-  1. Client software **SHOULD** use the sum of the block's priority fees or any other algorithm to determine `blockValue`.
\ No newline at end of file
+  1. Client software **SHOULD** use the sum of the block's priority fees or any other algorithm to determine `blockValue`.
```

Comment:
What is this line showing up in the diff?

Reply:
Could be extra white space at the end?

Reply:
yeah, this is whitespace as labeled in this commit

cf51b8954c95b2e61e0089ac4ab402d0624e3d4f

I'm happy to drop this commit if we don't like it